ipatests/test_integration/test_commands.py

```diff
@@ -1598,6 +1598,91 @@ def test_ipa_cacert_manage_prune(self):
+        tasks.kinit_admin(self.replicas[0])
+        tasks.kinit_admin(self.clients[0])
+        cmds = [
+            ['/usr/sbin/ipa-adtrust-install', '-a',
+             self.master.config.admin_password, '-U'],
+            ['/usr/sbin/ipa-replica-manage', 'del',
+             f"dummyhost.{self.master.domain.name}", '-p',
+             self.master.config.dirman_password],
+            ['/usr/sbin/ipa-csreplica-manage', 'del',
+             f"dummyhost.{self.master.domain.name}", '-p',
+             self.master.config.dirman_password],
+            ['/usr/sbin/ipa-kra-install', '-p',
+             self.master.config.dirman_password, '-U'],
+            ['/usr/sbin/ipa-server-certinstall', '-k', '--pin',
+             self.master.config.dirman_password, '-p',
+             self.master.config.dirman_password, paths.KDC_CERT,
+             paths.KDC_KEY]
+        ]
+        for cmd in cmds:
+            self.master.run_command(cmd, raiseonerr=False)
+            tasks.check_journal_does_not_contain_secret(
+                self.master, cmd[0]
+            )
+        for cmd in cmds:
+            self.replicas[0].run_command(cmd, raiseonerr=False)
+            tasks.check_journal_does_not_contain_secret(
+                self.replicas[0], cmd[0]
+            )
+        tasks.check_journal_does_not_contain_secret(
+            self.clients[0], 'python3'
+        )
+        # Backup and restore IPA and check secrets are not leaked.
+        backup_path = tasks.get_backup_dir(self.master)
+        restore_cmd = (
+            ['/usr/sbin/ipa-restore', '-p',
+             self.master.config.dirman_password,
+             backup_path, '-U']
+        )
+        self.master.run_command(restore_cmd)
+        tasks.check_journal_does_not_contain_secret(
+            self.master, restore_cmd[0]
+        )
+        # Checking for secrets in IPA server install
+        cmd = [
+            "/usr/sbin/ipa-server-install",
+            "-n",
+            self.master.domain.name,
+            "-r",
+            self.master.domain.realm,
+            "-p",
+            self.master.config.dirman_password,
+            "-a",
+            self.master.config.admin_password,
+            "-U",
+        ]
+        self.master.run_command(cmd, raiseonerr=False)
+        tasks.check_journal_does_not_contain_secret(
+            self.master, cmd[0]
+        )
+        # Checking for secrets in IPA replica install
+        cmd = [
+            "/usr/sbin/ipa-replica-install",
+            "-n",
+            self.master.domain.name,
+            "-r",
+            self.master.domain.realm,
+            "-p",
+            self.master.config.dirman_password,
+            "-w",
+            self.master.config.admin_password,
+            "-U",
+            "--server",
+            self.master.hostname
+        ]
+        self.replicas[0].run_command(cmd, raiseonerr=False)
+        tasks.check_journal_does_not_contain_secret(
+            self.replicas[0], cmd[0]
+        )
```

Comment:
Same comment as above, no need to call this command as it was already executed as part of the test setup. Replace lines 1667-1684 with
```
        tasks.check_journal_does_not_contain_secret(
            self.replicas[0], "/usr/sbin/ipa-replica-install"
        )
```